apps/prairielearn/src/pages/partials/question.ejs

```diff
@@ -218,7 +218,7 @@
   <div id="copyQuestionModal" class="modal fade" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
     <div class="modal-dialog">
       <div class="modal-content">
-        <form method="POST">
+        <form method="POST" action="<%= question_copy_targets[0].copy_url %>">
```

Comment:
Are we guaranteed that `question_copy_targets` isn't the empty list? I see that we check it's not `undefined` above, but not that it's length is more than zero.

Reply:
Given that a lot of tests failed, you're 100% right that this is an issue 🙂 I'll fix it!

Reply:
This should be addressed by e491d7b.